components/content-service-api/usage.proto

```diff
@@ -11,6 +11,9 @@ option go_package = "github.com/gitpod-io/gitpod/content-service/api";
 service UsageReportService {
   // UploadURL provides a URL to which clients can upload the content via HTTP PUT.
   rpc UploadURL(UsageReportUploadURLRequest) returns (UsageReportUploadURLResponse) {}
+
+  // GetDownloadURL retrieves a URL which can be used to download a Usage Report.
+  rpc GetDownloadURL(GetDownloadURLRequest) returns (GetDownloadURLResponse) {}
```

Comment:
Should we have a verb `Get` for DownloadURL? Because we use UploadURL at line 13, should we define it as `DownloadURL` instead?

Reply:
I don't mind. IMO, the RPC above should've had it as it indicates the action being performed but I can drop it from here also..

Reply:
References
- https://github.com/gitpod-io/gitpod/blob/b5a7a2ff91f97e78852e44bab4f54fd7aeb433da/components/content-service-api/blobs.proto#L11-L16
- https://github.com/gitpod-io/gitpod/blob/b5a7a2ff91f97e78852e44bab4f54fd7aeb433da/components/content-service-api/ideplugin.proto#L11-L16

Reply:
I've updated the protos.